.github/workflows/build-test-ubuntu-core-full_node.yml

```diff
@@ -97,14 +97,9 @@ jobs:
     - name: Install developer requirements
       run: |
         . ./activate
-        venv/bin/python -m pip install pytest pytest-asyncio pytest-xdist pytest-monitor
```

Comment:
it looks like you dropped this dependency, along with the code using it below. Are these files generated and I added code to it manually?

Reply:
Yes, these are generated - I will add a warning to the generated files